Tests/configure_and_test_integration_instances.py

```diff
@@ -1258,13 +1257,14 @@ def test_pack_zip(content_path, target):
             if not test_path.endswith('.yml'):
                 continue
             test = test.name
+
+            if not (test.startswith('playbook-') or test.startswith('script-')):
+                test_type = find_type(test_path).value
```

Comment:
I think find_type finds test playbooks and scripts as a separate file type... please check that

Reply:
@guyfreund What is the meaning of this term?